libs/blocks/aside/aside.js

```diff
@@ -31,8 +31,14 @@ const blockConfig = {
     [medium]: ['s', 's'],
     [large]: ['l', 'm'],
   },
+  promobar: {
+    'mobile-only': ['s', 's'],
+    'tablet-only': ['s', 's'],
+    'desktop-only': ['m', 'l'],
+  },
 };
 const FORMAT_REGEX = /^format:/i;
+const mediaports = ['mobile-only', 'tablet-only', 'desktop-only'];
```

Comment:
could avoid config with:
```
const mediaports = {
    'mobile-only': ['s', 's'], 
    'tablet-only': ['s', 's'],
    'desktop-only': ['m', 'l']
};
// or 
const mediaports = {
    'responsive-only': ['s', 's'], 
    'desktop-only': ['m', 'l']
};
```

Reply:
I had added the config to block config as it contains the styling of all aside formats. Since this style was specific to this function of promobar I have moved it inside combineTextBocks function and we will not be requiring this in global definition.